project-communications/participation-communication-strategy.md

```diff
@@ -0,0 +1,131 @@
+# Participation and Communication Strategy
+
+This is where you will find all information and strategic direction of our communicaion and engagement to ensure participation on the AutSPACEs Project. 
```

Comment:
Very clear and great to have this all in one place! Maybe it would be best to have it as a folder in the main repo?